doc/user/content/sql/create-role.md

```diff
@@ -22,20 +22,39 @@ _role_name_         | A name for the role.
 
 ## Details
 
-Unlike PostgreSQL, Materialize derives the `LOGIN` and `SUPERUSER`
-attributes for a role during authentication, every time that role tries
-to connect. Therefore, you cannot specify either
-attribute when creating a new role. Additionally, we do not support the
-`CREATE USER` command, because it implies a `LOGIN` attribute for the role.
+Unlike PostgreSQL,
 
-Unlike PostgreSQL, Materialize does not currently support `NOINHERIT`.
+- Materialize does not support the `LOGIN` and `SUPERUSER` attributes for
```

Comment:
I'm of two minds about the phrasing.  I almost want to ignore "Unlike PostgreSQL" ... and just go:

`CREATE ROLE` does not support ...

And only for the related commands that aren't supported, just state Materialize does not support ... but, I went back and forth.

Reply:
can we break the bullets down by role attribute (like `INHERIT`) instead? so something like:

> `INHERIT`
> - Materialize implicitly uses `INHERIT` when the `CREATE ROLE` command is executed. That is, `CREATE ROLE <name>` and `CREATE ROLE <name> WITH INHERIT` are equivalent.
> - Materialize does not support setting the `INHERIT` role attribute directly.
>
> `LOGIN`
> - Materialize does not support setting the `LOGIN` role attribute directly. Instead, Materialize derives this attribute for a role during authentication each time the role tries to connect.
> - Materialize does not support the `CREATE USER` command, as the command implies a `LOGIN` attribute for the role.

etc.

Reply:
> Materialize does not support the CREATE USER command, as the command implies a LOGIN attribute for the role.

also, not sure if this should have its own page instead!

Reply:
Agree.  We could have some kind of an appendix page where we specify what we don't support, but will hold off on creating that page for when we have a decent enough list.  Then, we can mention the blurb about not supporting create user here and have them go to that page (if they want to see what else we might not support).